addOns/quickstart/src/main/javahelp/org/zaproxy/zap/extension/quickstart/resources/help/contents/quickstart.html

```diff
@@ -8,28 +8,65 @@
 </HEAD>
 <BODY BGCOLOR="#ffffff">
 <H1>Quick Start</H1>
+
 <p>
-	The Quick Start tab provides you with an easy way to quickly test a web application.
-</p>
-<p>
-	Enter the URL of your target application and press the 'Attack' button.
-</p>
-<p>
-	ZAP will spider the target and then attack it with the active scanner.
-</p>
-<p>
-	For a more in depth test you should explore your application using your browser or 
-	automated regression tests while proxying through ZAP.
+	The Quick Start tab provides you with an easy way to quickly test a web application.<br>
+	The top screen shows 3 buttons that take you to the following screens.
 </p>
-<H2>Launch Browser</H2>
-As long as you have the Selenium add-on installed then you will see an option to launch browsers.<br>
+
+<H2>Automated Scan</H2>
+
+The Automated Scan screen allows you to launch an automated scan against an application - just enter its URL below and press 'Attack'.
+<br>
+Please be aware that you should only attack applications that you have been specifically been given permission to test.
+
+<H3>URL to attack</H3>
+
+This is the URL of the application you wish to attack, starting with 'http://' or 'https://'.<br>
+ZAP must be able to access this URL for the attack to proceed.
+
+<H3>Use traditional spider</H3>
+
+The traditional spider explores the application by finding links in HTML pages. 
+It is very fast but cannot handle applications which make heavy use of JavaScript. 
+
+<H3>Use ajax spider</H3>
+
+The ajax spider explores the application by launching the browser you have chosen and clicking on the links it finds. 
+Is is slower than the traditional spider but handles JavaScript well.<br>
+This option is only shown if the ajax spider add-on is installed. 
+If it is not available then you can download and install it for free from the ZAP Marketplace.
+
+<H2>Manual Explore</H2>
+
+The Manual Explores screen allows you to launch the browser of your choice so that you can explore your application while proxying through ZAP.
```

Comment:
Explores doesn't need to be plural.

Note it ignores cert issues and disables browser security controls?